mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -1420,7 +1425,8 @@ func ResourceContainerCluster() *schema.Resource {
 			"master_authorized_networks_config": {
 				Type:        schema.TypeList,
 				Optional:    true,
-				Computed:    true,
+				Deprecated:    "use control_plane_endpoints_config.ip_endpoints_config.authorized_networks_config instead",
```

Comment:
Could you add an update test that performs the migration required by this deprecation from one step to another? Ideally backwards from the new to old option as well- we should support both directions in `6.X`.

Reply:
I've added separate tests for private nodes (network_config field), master_authorized_networks_config, misc private_cluster_config fields, and DNS enablement. I haven't run it yet, since it takes a while to run, I'll double check that soon.